Modules/Core/Common/include/itkVector.h

```diff
@@ -328,6 +328,26 @@ swap(Vector<T, NVectorDimension> & a, Vector<T, NVectorDimension> & b)
   a.swap(b);
 }
 
+
+/** Makes a Vector object, having the specified values as coordinates. */
+template <typename TValue, typename... TVariadic>
+auto
+MakeVector(const TValue firstValue, const TVariadic... otherValues)
+{
+  // Assert that the other values have the same type as the first value.
+  const auto assertSameType = [](const auto value) {
+    static_assert(std::is_same<decltype(value), const TValue>::value, "Each value must have the same type!");
+    return true;
+  };
+  const bool assertions[] = { true, assertSameType(otherValues)... };
+  (void)assertions;
+  (void)assertSameType;
```

Comment:
I found it a bit disappointing that it took me so much code to simply just assert that all values have the same type, but for now I think it's good enough. Approved 👍